# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/BackgroundChildImpl.h
# Commit: 0868fb4018a7
# Full Hash: 0868fb4018a7ae2002b8f2fcfd88643c2e060f07
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-11 09:21:34
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 2 - CookieStore API - IPDL, r=edgul,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215145
# ==============================================================================

diff -r 9c8b6e35bf3f -r 0868fb4018a7 ipc/glue/BackgroundChildImpl.h
--- a/ipc/glue/BackgroundChildImpl.h	Tue Sep 10 16:31:38 2024 +0000
+++ b/ipc/glue/BackgroundChildImpl.h	Tue Sep 10 16:31:38 2024 +0000
@@ -125,6 +125,10 @@
   virtual bool DeallocPBroadcastChannelChild(
       PBroadcastChannelChild* aActor) override;
 
+  virtual PCookieStoreChild* AllocPCookieStoreChild() override;
+
+  virtual bool DeallocPCookieStoreChild(PCookieStoreChild* aActor) override;
+
   virtual PServiceWorkerManagerChild* AllocPServiceWorkerManagerChild()
       override;
 